### ISSUE TITLE
arx forecaster: first lag "duplicates" fits in bizzare ways

### ISSUE THREAD

**Issue by dsweber2** [cmu-delphi/epipredict#209]
when the first lags is larger than zero, the output is duplicated that many times +1
```r
jhu <- case_death_rate_subset
jhu <- jhu %>% filter(time_value >= max(time_value) - 30)
out0 <- arx_forecaster(
  jhu,
  outcome = "death_rate",
  predictors = c("case_rate", "death_rate"),
  args_list = arx_args_list(
  lags = c(0))
)
out1 <- arx_forecaster(
  jhu,
  outcome = "death_rate",
  predictors = c("case_rate", "death_rate"),
  args_list = arx_args_list(
  lags = c(1))
)
out2 <- arx_forecaster(
  jhu,
  outcome = "death_rate",
  predictors = c("case_rate", "death_rate"),
  args_list = arx_args_list(
  lags = c(2))
)
```

**Comment by dajmcdon** [comment]
This is due to #207

**Comment by rachlobay** [comment]
By output, you mean when we look at the preds? Such as `out2$predictions %>% filter(geo_value == "ak")` returning duplicate rows? If so, I think this may now be fixed with PR #211.

**Comment by dsweber2** [comment]
Glad to hear it! Yes, that is what I mean, and it does sound like it's downstream of what you're working on. If I understand correctly, if you add something like "closes #209" to the first message in your pull request, you can get this issue to [auto-close](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) when that merges.
